tests/functional/pages/surveys/census/communal/completion-preference-establishment.page.js

```diff
@@ -9,12 +9,12 @@ class CompletionPreferenceEstablishmentPage extends MultipleChoiceWithOtherPage
   }
 
   clickCompletionPreferenceEstablishmentAnswerOnline() {
-    browser.element('[id="completion-preference-establishment-answer-1"]').click()
+    browser.element('[id="completion-preference-establishment-answer-0"]').click()
```

Comment:
we should probably create a card to investigate generating these on the fly to keep @benjefferies happy :)